vite.config.ts

```diff
@@ -0,0 +1,7 @@
+import { defineConfig } from 'vite'
```

Comment:
**praise**: Good job on transitioning to TypeScript for the Vite configuration. This enhances type safety and maintainability.